forte/data/data_store.py

```diff
@@ -533,9 +552,20 @@ def next_entry(self, tid: int) -> List:
             tid (int): Unique id of the entry.
 
         Returns:
-            The next entry of the same type as the ``tid`` entry.
+            A list of attributes representing the next entry of the same type
+            as the ``tid`` entry.
+
+        Raises:
+            IndexError: An error occurred accessing the next entry of the last
+                element in entry list.
         """
-        raise NotImplementedError
+        _, entry_type, index_id = self.get_entry(tid=tid)
```

Comment:
The first return value `_` actually contains entry information right? Why do we need to discard it and use the `index_id` again. @wanglec @qinzzz

Reply:
`index_id` is not in the entry's list, and next_entry only need entry's index and type informaton.

Reply:
we should call another `get_entry_index` function (mentioned above) instead of this.

Reply:
So `get_entry_index` will only return the `index_id`, right?

Reply:
yes, so `get_entry` from `tid` directly get the value, while `get_enty_index` can provide an index for `delete`, `next` purposes.